packages/cli/src/License.ts

```diff
@@ -94,11 +100,22 @@ export class License {
 	}
 
 	async collectUsageMetrics() {
```

Comment:
instead of keeping adding unrelated code to `License`, we should extract this method into a separate `UsageMetricsService`.